librina/src/netlink-parsers.h

```diff
@@ -612,6 +611,39 @@ int putSecurityManagerConfigurationObject(nl_msg* netlinkMessage,
 
 SecurityManagerConfiguration * parseSecurityManagerConfigurationObject(nlattr *nested);
 
+/* AuthSDUProtectionProfile CLASS */
+enum AuthSDUProtectionProfileAttributes {
+    AUTHP_AUTH_POLICY = 1,
+    AUTHP_ENCRYPT_POLICY,
+    AUTHP_TTL_POLICY,
+    AUTHP_CRC_POLICY,
+    __AUTHP_ATTR_MAX,
+};
+
+#define AUTHP_ATTR_MAX (__AUTHP_ATTR_MAX -1)
+
+int putAuthSDUProtectionProfile(nl_msg* netlinkMessage,
+			   	const AuthSDUProtectionProfile& object);
+AuthSDUProtectionProfile * parseAuthSDUProtectionProfile(nlattr *nested);
+
+enum SpecificAuthSDUProtectionProfileAttributes {
+    SAUTHP_UNDER_DIF = 1,
+    SAUTHP_AUTH_PROFILE,
+    __SAUTHP_ATTR_MAX,
+};
+
+#define SAUTHP_ATTR_MAX (__SAUTHP_ATTR_MAX -1)
+
+int putSpecificAuthSDUProtectionProfile(nl_msg* netlinkMessage,
+					const std::string& under_dif,
+			   	        const AuthSDUProtectionProfile& object);
+int parseSpecificSDUProtectionProfile(nlattr *nested,
```

Comment:
No, this function parses a  single specific SDU protection profile and adds it to the map.